deps.bzl

```diff
@@ -6935,3 +6943,15 @@ def install_static_dependencies(workspace_name = "buildbuddy"):
         sha256 = "85b1c2591274422234955e906aee39e6f793a88a74f3efc49a1852a0646ce08f",
         urls = ["https://storage.googleapis.com/buildbuddy-tools/binaries/clang-format/clang-format-14_linux-x86_64"],
     )
+
+    http_file(
+        name = "com_github_buildbuddy_io_podman_static_podman-linux-amd64",
+        urls = ["https://github.com/buildbuddy-io/podman-static/releases/download/buildbuddy-{podman_version}/podman-linux-amd64.tar.gz".format(podman_version = PODMAN_VERSION)],
+        sha256 = PODMAN_STATIC_SHA256_AMD64,
+    )
+
+    http_file(
+        name = "com_github_buildbuddy_io_podman_static_podman-linux-arm64",
+        urls = ["https://github.com/buildbuddy-io/podman-static/releases/download/buildbuddy-{podman_version}/podman-linux-arm64.tar.gz".format(podman_version = PODMAN_VERSION)],
+        sha256 = PODMAN_STATIC_SHA256_ARM64,
+    )
```

Comment:
Question: I wonder if it would be better if we use `http_archive` with a simple inline `build_file_content` here?

That would let us unpack the archive at the repository phase and use only the static Go binary during the build phase. In such a case, we would not have to worry about the README or `registries.conf` in the existing archive at all.

Reply:
I guess we still do care whether the registries.conf exists in the right path eh? 🤔

Reply:
yeah registries.conf comes from our repro so IDK if build_file_content would work